app/assets/stylesheets/trustsfrontend-app.scss

```diff
@@ -169,6 +169,7 @@
 }
 
 .agentClientRef, .declarationReferenceNumber {font-weight:700;}
+.form-control.agent-internal-reference {width: 100% !important;}
```

Comment:
Is there an alternative that we can do here? Do we need to absolutely use !important. Always shouts that our selectivity isn't quite correct.

Reply:
Also, curious why this needs to be 100%?

Reply:
Removed !important (was an oversight  and not required).
There is a requirement to show all 56 characters for a client ref. By default input["text"] is set to a width of 50% within its parent container (min-width 641px), hence setting this to 100% will allow the user to see the complete max input?

Reply:
Is there not an alternative css class we can set to expand the width or is this the largest available?